moment-timezone.js

```diff
@@ -330,6 +350,16 @@
 		}
 	}
 
+	function findYearLike(year) {
+		for (var j = 2038; j > 2000; j--) {
+			if (new Date(year + '-01-01').getDay() === new Date(j + '-12-31').getDay() && 
```

Comment:
You're repeating the same check here instead of comparing the first day of the week of the target year with the last day of the week of the candidate year, instead of comparing the first and last days of the week of the target year with those of the candidate year.

Reply:
fixed that already, will push changes tomorrow

Reply:
Any news about the fix @ellenaua ? :)